### PR TITLE
Purview GA Managment SDK

### PR DESCRIPTION
**Pull request by annoakes**
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [x] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/main/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running `generate.ps1`. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version.
	
	**Note: We have recently updated the PSH module called by `generate.ps1` to emit additional data. This would help reduce/eliminate the Code Verification check error. Please run following command**:

	    `dotnet msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules`

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

**Comment by annoakes** [comment]
Swagger review: https://github.com/Azure/azure-rest-api-specs/pull/15245

File: sdk/purview/Microsoft.Azure.Management.Purview/src/Microsoft.Azure.Management.Purview.csproj [hunk 1]
```diff
@@ -5,11 +5,11 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Purview</PackageId>
     <Description>Azure Purview is a unified data governance service that helps you manage and govern your on-premises, multicloud, and software-as-a-service (SaaS) data. </Description>
-    <Version>1.0.0-beta.1</Version>
+    <Version>1.0.0.1</Version>
```
Comment (allenjzhang) [inline sdk/purview/Microsoft.Azure.Management.Purview/src/Microsoft.Azure.Management.Purview.csproj:8]: The version number should 1.0.1.0. The assemblyinfo as well.

**Review by allenjzhang — CHANGES_REQUESTED** [review]
Is your API version also changing? If so, please update AzSDK.RP.props file.

File: sdk/purview/Microsoft.Azure.Management.Purview/src/Microsoft.Azure.Management.Purview.csproj (hunk 1 above)
Comment (annoakes) [reply sdk/purview/Microsoft.Azure.Management.Purview/src/Microsoft.Azure.Management.Purview.csproj]: updated

**Comment by annoakes** [comment]
done

---
In reply to: [712264102](https://github.com/Azure/azure-sdk-for-net/pull/22778#issuecomment-712264102) [](http://example.com/codeflow?ancestors=712264102,712264102)

File: sdk/purview/Microsoft.Azure.Management.Purview/src/Generated/Models/TrackedResource.cs [hunk 2]
```diff
@@ -38,14 +38,17 @@ public TrackedResource()
         /// resource</param>
         /// <param name="location">Gets or sets the location.</param>
         /// <param name="name">Gets or sets the name.</param>
+        /// <param name="systemData">Metadata pertaining to creation and last
+        /// modification of the resource.</param>
         /// <param name="tags">Tags on the azure resource.</param>
         /// <param name="type">Gets or sets the type.</param>
-        public TrackedResource(string id = default(string), Identity identity = default(Identity), string location = default(string), string name = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string type = default(string))
```
Comment (allenjzhang) [inline sdk/purview/Microsoft.Azure.Management.Purview/src/Generated/Models/TrackedResource.cs:43]: ctor breaking change. Please provide old ctor in customization code.

File: sdk/purview/Microsoft.Azure.Management.Purview/src/Generated/Models/Account.cs [hunk 3]
```diff
@@ -52,27 +54,30 @@ public Account()
...
-            : base(id, identity, location, name, tags, type)
+        public Account(string id = default(string), Identity identity = default(Identity), string location = default(string), string name = default(string), TrackedResourceSystemData systemData = default(TrackedResourceSystemData), IDictionary<string, string> tags = default(IDictionary<string, string>), string type = default(string), CloudConnectors cloudConnectors = default(CloudConnectors), System.DateTime? createdAt = default(System.DateTime?), string createdBy = default(string), string createdByObjectId = default(string), AccountPropertiesEndpoints endpoints = default(AccountPropertiesEndpoints), string friendlyName = default(string), string managedResourceGroupName = default(string), AccountPropertiesManagedResources managedResources = default(AccountPropertiesManagedResources), IList<PrivateEndpointConnection> privateEndpointConnections = default(IList<PrivateEndpointConnection>), string provisioningState = default(string), string publicNetworkAccess = default(string), AccountSku sku = default(AccountSku))
```
Comment (allenjzhang) [inline sdk/purview/Microsoft.Azure.Management.Purview/src/Generated/Models/Account.cs:71]: ctor breaking change. Please provide old ctor in customization code.

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**